src/ui/Debug/DebugHeader.ts

```diff
@@ -60,6 +60,7 @@ export class DebugHeader {
   private handleDoneBuildingQuery(args: IDoneBuildingQueryEventArgs) {
     args.queryBuilder.enableDebug = this.debug || args.queryBuilder.enableDebug;
     if (this.requestAllFields) {
+      args.queryBuilder.fieldsToInclude = undefined;
```

Comment:
As soon as fieldsToInclude is defined (even as an empty list), the option is added to the queryBuilder and returns no result for the debug panel.